app/lib/hpitt.rb

```diff
@@ -176,10 +255,38 @@ def to_nationality_ids(raw_string)
       Nationality.where(name: raw_string&.downcase).ids
     end
 
+    def to_post_code(raw_string)
+      return if raw_string.blank?
+
+      UKPostcode.parse(raw_string.gsub(/\W/, "")).to_s
+    end
+
     def parse_date(raw_date)
       return if raw_date.blank?
 
       Date.parse(raw_date)
     end
+
+    def normalise_string(raw_string)
+      raw_string
+      .downcase
+      .gsub(/\(.*\)/, "")
+      .split
+      .reject { |word| REJECTED_WORD_LIST.include? word }
+      .join(" ")
+      .gsub(/[^\w]/, "")
```

Comment:
something for the future - we should probably label the regexs so it's easier to mentally parse, we've got quite a few in here